src/jsonLogic.js

```diff
@@ -106,3 +106,33 @@ export function yupSchemaWithCustomJSONLogic({ field, validations, config, id })
       }
     );
 }
+
+export function calculateComputedAttributes(fieldParams, { parentID = 'root' } = {}) {
+  return ({ validations, formValues }) => {
+    const { name, computedAttributes } = fieldParams;
+    const attributes = Object.fromEntries(
+      Object.entries(computedAttributes)
+        .map(handleComputedAttribute(validations, formValues, parentID, name))
+        .filter(([, value]) => value !== null)
+    );
+
+    return attributes;
+  };
+}
+
+function handleComputedAttribute(validations, formValues, parentID, name) {
+  return ([key, value]) => {
+    if (key === 'const')
+      return [
+        key,
+        validations.getScope(parentID).evaluateComputedValueRuleForField(value, formValues, name),
```

Comment:
Looking at the signature of the `evaluateComputedValueRuleForField` it appears that the last parameter, `name`, is not expected.

Reply:
this will come soon (error handling) but i'll remove for now

Reply:
Updated